AirCasting/SDSync/ViewsFlow/SDSyncingAB/SyncingABView.swift

```diff
@@ -5,76 +5,65 @@
 //  Created by Lunar on 04/02/2021.
 //
 
-import CoreBluetooth
 import SwiftUI
 
 struct SyncingABView<VM: SDSyncViewModel>: View {
-    
     @Environment(\.presentationMode) var presentationMode
     @StateObject var viewModel: VM
-    let baseURL: BaseURLProvider
     @Binding var creatingSessionFlowContinues: Bool
-    @State private var showNextScreen: Bool = false
-    @State private var presentAlert: Bool = false
-    @Binding var sdSyncContinues: Bool
-    
     
     var body: some View {
-        VStack(spacing: 50) {
-            ProgressView(value: 0.5)
+        VStack(alignment: .leading, spacing: 40) {
+            ProgressView(value: 0.7)
+            Spacer()
             ZStack(alignment: Alignment(horizontal: .trailing, vertical: .bottom), content: {
-                Image("airbeam")
-                    .resizable()
-                    .aspectRatio(contentMode: .fit)
+                syncingImage
                 loader
                     .padding()
                     .padding(.vertical)
             })
-            
+            Spacer()
             VStack(alignment: .leading, spacing: 15) {
                 titleLabel
                 messageLabel
             }
-        }.background(
-            NavigationLink(
-                destination: completeView,
-                isActive: $showNextScreen,
-                label: {
-                    EmptyView()
-                }
-            )
-        )
+            Spacer()
+        }
         .padding()
+        .background(navigationLink)
         .onReceive(viewModel.isSyncCompleted, perform: { isConnected in
-            showNextScreen = isConnected
+            viewModel.presentNextScreen = isConnected
```

Comment:
same as with others - let's make it into a func on VM that does that write instead.